integration/templates/nuxt-node/pages/user.vue

```diff
@@ -0,0 +1,11 @@
+<script setup>
+const { data: user } = await useFetch('/api/me');
```

Comment:
[`useFetch`](https://nuxt.com/docs/api/composables/use-fetch) makes API calls on the server during initial page load, and on the client during subsequent navigation